src/lib/mixpanel/helpers.ts

```diff
@@ -1,8 +1,7 @@
 import type { AssetId } from '@shapeshiftoss/caip'
-import type { Asset } from '@shapeshiftoss/types'
+import type { Asset, AssetsById } from '@shapeshiftoss/types'
```

Comment:
This was partial before
```suggestion
import type { Asset, PartialAssetsById } from '@shapeshiftoss/types'
```